examples/with-mobx/components/Clock.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { inject, observer } from 'mobx-react'
+
+const Clock = inject('store')(observer(({store}) => {
```

Comment:
In my latest commit fa87184 I fixed it by having a page component that does the injection, and calls the Clock component